src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -660,6 +663,7 @@ ApiGlobe.prototype.setRange = function setRange(pRange /* , bool anim*/) {
     eventRange.oldRange = this.getRange();
     loaded = false;
     this.scene.currentControls().setRange(pRange);
+    this.viewerDiv.dispatchEvent(eventRange);
```

Comment:
So I guess we should change that for all dispatched events. Maybe in an another PR ? WDYT ?